cf-deploy-teardown/tasks/k8s-ready-state-check.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+#Script to determine is the K8s host is "ready" for cf deployment
+
+set -ex
+
+# cgroup memory & swap accounting in /proc/cmdline
+
+grep -w "cgroup_enable=memory" /proc/cmdline
+echo "Verified: cgroup_enable memory"
+
+grep -w "swapaccount=1" /proc/cmdline
+echo "Verified: swapaccount enabled"
+
+# docker info should show overlay2
+
+docker info | grep -w "Storage Driver: overlay2"
+echo "Verified: 'docker info should show overlay2'"
+
+# kube-dns shows 4/4 ready
+
+kube_dns=$(k get pods : | grep "kube-dns-")
+if [[ $kube_dns == *"4/4 Running"* ]]; then
+  echo "Verified: 'kube-dns shows 4/4 ready'"
+fi
+
+# ntp is installed and running
+
+systemctl status ntpd| grep -w "Active: active (running)"
+echo "Verified: ntp is running"
```

Comment:
nope. its for K8 host, Aaron said it does no matter on Vagrant box. So i just used `sudo systemctl enable ntpd; sudo systemctl start ntpd` to test this